src/components/existing-request/existing-request-display.vue

```diff
@@ -533,62 +523,75 @@ export default class ExistingRequestDisplay extends Mixins(
   private actionText (action: NrAction): string {
     switch (action) {
       case NrAction.CANCEL: return 'Cancel Name Request'
-      case NrAction.REAPPLY: return 'Renew Name Request ($30)'
+      case NrAction.RENEW: return 'Renew Name Request ($30)' // FUTURE: fetch this fee
       case NrAction.RECEIPTS: return 'Download Receipts'
       case NrAction.REQUEST_REFUND: return 'Cancel and Refund'
       case NrAction.RESEND: return 'Resend Email' // FUTURE: will be removed
-      case NrAction.RESULT: return 'Download Results' // FUTURE: will be implemented
+      case NrAction.RESULT: return 'Download Results'
+      case NrAction.RESUBMIT: return 'Resubmit Name Request ($30)'
       case NrAction.RETRY_PAYMENT: return 'Retry Payment'
-      case NrAction.UPGRADE: return 'Upgrade Priority ($100)'
+      case NrAction.UPGRADE: return 'Upgrade Priority ($100)' // FUTURE: fetch this fee
       default: return this.toTitleCase(action)
     }
   }
 
   private async handleButtonClick (action: NrAction) {
+    // FUTURE: reinstate this check?
     // const confirmed = await newReqModule.confirmAction(action)
     const confirmed = true
+
     if (confirmed) {
```

Comment:
not sure i understand the purpose of this const.

Reply:
You're picking on code I didn't write :)

It looks to me like this const previously confirmed the action before handling the button click, but at some point was disabled. The const was left in place to minimize code changes. I don't know if this is a temporary or permanent change. That's why I added the FUTURE comment on whether to reinstate this check.

Reply:
hahah sorry sorry, i think we are both guilty of that! 

No prob, no action required :)